proto/v1/upgrade.proto

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grafeas.v1;
+
+option go_package = "github.com/grafeas/grafeas/proto/v1/grafeas_go_proto";
+option java_multiple_files = true;
+option java_package = "io.grafeas.v1";
+option objc_class_prefix = "GRA";
+
+import "proto/v1/package.proto";
+
+// An upgrade note represents a potential upgrade of a package to a given version.
+// In general there should be an upgrade note for every single package version combination.
+message UpgradeNote {
+  // Required - The package this Upgrade is for.
+  string package = 1;
+  // Required - The version of the package in machine + human readable form,
+  grafeas.v1.Version version = 2;
+  // Per operating system metadata.
+  repeated UpgradeDistribution distributions = 3;
+}
+
+// The Upgrade Distribution represents per cpe metadata for a given upgrade. Some distributions
+// have additional metadata around updates, classifying them into various string categories and
+// severities.
+message UpgradeDistribution {
+  // Required - The specific operating system this metadata applies to.
```

Comment:
please add: ", e.g. <any os example>"

Reply:
Added a link to match vulnerability.proto.